### DISCUSSION TITLE
Vue SFC Tailwindcss Not Working

### DISCUSSION THREAD

**Discussion by kirillpanfile** [zed-industries/zed#7633] [Feature Requests]
Hi, I tried working with Tailwind CSS in HTML and JSX/TSX, and everything works great. However, it doesn't work in Vue Single File Components (SFC). I tried adding `lang="pug"` or `lang="html"` to the root template, but Tailwind CSS doesn't work. Is there a fix or some guide on how to make it work?

**Comment by davidwebca** [comment]
Have you figured it out? I just stumbled onto the same issue.

**Reply by davidwebca** [reply]
Oh! I managed to do something while fiddling. Since I'm using eslint, that's what I had to output. Vue.js is the language identifier and not "vue" or "Vue". Unless in tailwindcss-language-server configs. I've got it to work with the following : 

```json
"languages": {
      "Vue.js": {
          "formatter": {
              "code_actions": {
                  "source.fixAll.eslint": true
              }
          }
      }
  },
  "lsp": {
      "tailwindcss-language-server": {
          "settings": {
              "includeLanguages": {
                  "blade": "html",
                  "vue": "html"
              }
          }
      }
  }
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
